go.mod

```diff
@@ -4,6 +4,7 @@ go 1.24.2
 
 require (
 	cloud.google.com/go/compute/metadata v0.7.0
+	github.com/AliyunContainerService/ack-ram-tool/pkg/ecsmetadata v0.0.7
```

Comment:
Is this library an absolute must to have? If yes, we may need someone from alibabacloud to confirm where this is the recomended/safe to use library.

Reply:
@ivankatliarchuk I'm from Alibaba Cloud, but I'm not sure how to prove it to you. Any suggestions?

Reply:
Any chance to move this library under https://github.com/aliyun ?

Reply:
@ivankatliarchuk Unfortunately, that isn't an easy task. Anyway, I'll write a simple replacement code instead of using this package.